pkg/sources/git/git.go

```diff
@@ -443,9 +440,8 @@ func (s *Git) gitChunk(diff gitparse.Diff, fileName, email, hash, when, urlMetad
 			}
 		}
 
-		_, err := newChunkBuffer.Write(line)
-		if err != nil {
-			log.WithError(err).Error("Could not write line to git diff buffer.")
+		if _, err := newChunkBuffer.Write(line); err != nil {
+			ctx.Logger().V(1).Info("error writing to chunk buffer", "error", err, "filename", fileName, "commit", hash, "file", diff.PathB)
 		}
```

Comment:
Here we could use

```go
ctx.Logger().Error(err, "error writing to chunk buffer", "filename", fileName, ...)
```

which will be an error that gets sent to sentry. I think in this case it makes sense to do it how you have here though.

Reply:
Ahh good to know, I was unaware of that actually.